lib/utils_input.go

```diff
@@ -0,0 +1,129 @@
+package lib
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type INPUT_TYPE string
+
+const (
+	INPUT_TYPE_IP       INPUT_TYPE = "IP"
+	INPUT_TYPE_IP_RANGE INPUT_TYPE = "IPRange"
+	INPUT_TYPE_CIDR     INPUT_TYPE = "CIDR"
+	INPUT_TYPE_FILE     INPUT_TYPE = "File"
```

Comment:
Files are a source, not an input type: the generic functionality, if told to read from files by the user, should do so by opening them and parsing their inputs for ips/ranges/cidrs/asns/etc. The same way it transparently 'opens' the 'stdin file' and reads from it when requested.